hw02/PlayerController.cs

```diff
@@ -34,25 +56,61 @@ void Update()
         // TODO: Problem 4: What is an if statement? Write a couple of if statements to move the player with the WASD keys. 
         // We want to transform the player's position along the vertical and horizontal axis then multiply it by 
         // the corresponding variables we defined earlier. Lastly, we will need to multiply it all by Time.DeltaTime
+
+        // Check if the player is pressing the W key for forward movement
+        if (Input.GetKey(KeyCode.W))
+        {
+            transform.Translate(Vector3.forward * movementSpeed * Time.deltaTime);
+        }
+
+        // Check if the player is pressing the S key for backward movement
+        if (Input.GetKey(KeyCode.S))
+        {
+            transform.Translate(Vector3.back * movementSpeed * Time.deltaTime);
+        }
+
+        // Check if the player is pressing the A key for left movement
+        if (Input.GetKey(KeyCode.A))
+        {
+            transform.Translate(Vector3.left * movementSpeed * Time.deltaTime);
+        }
+
+        // Check if the player is pressing the D key for right movement
+        if (Input.GetKey(KeyCode.D))
+        {
+            transform.Translate(Vector3.right * movementSpeed * Time.deltaTime);
+        }
+
+
     }
 
     // TODO: Problem 5: What do the lines here do? Please provide inline comments and describe what each line is doing
-    // What do Header, SerializeField, and Tooltip mean? 
-    [Header("General Setup Settings")]
-    [SerializeField] private InputAction movement;
-    [Tooltip("How fast player moves up and down based upon player input")][SerializeField] float controlSpeed = 30f;
-    [Tooltip("How far player moves horizontally")][SerializeField] float xRange = 10f;
-    [Tooltip("How far player moves vertically")][SerializeField] float yRange = 10f;
+
+    [Header("General Setup Settings")] // Adds a section header in the Unity Inspector for better organization
+    [SerializeField] private InputAction movement; // Allows private 'movement' InputAction to be modified in the Unity Inspector
+    [Tooltip("How fast player moves up and down based upon player input")][SerializeField] float controlSpeed = 30f; // Displays a tooltip in the Inspector and control speed of movement along the vertical axis
+    [Tooltip("How far player moves horizontally")][SerializeField] float xRange = 10f; // Tooltip describing the xRange variable and SerializeField maximum horizontal movement range
+    [Tooltip("How far player moves vertically")][SerializeField] float yRange = 10f;  // Tooltip describing the yRange variable and SerializeField maximum vertical movement range
```

Comment:
Correct. Those are property attributes that will help others understanding the variables in the inspector.